src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -196,7 +196,7 @@ contract SecurityCouncilManager is
 
     function _swapMembers(address _addressToRemove, address _addressToAdd)
         internal
-        returns (Cohort cohort)
+        returns (Cohort)
```

Comment:
why this change? just out of curiosity

Reply:
mmm, this one I'm not sure. There was another case where the var name was being mirroed, but I dont think this was it.

Generally I prefer this method - since it keeps the return statement explicit. But I'm not strict about it :)